client/src/App.css

```diff
@@ -111,7 +111,16 @@ section.card div {
 .questionNum {
   font: 50px 'Helvetica', sans-serif;
 }
+
 .overlay {
   color: #bfbfbf;
   padding-left: 2px;
 }
+
+.def_glyph {
```

Comment:
@sdee dashes are preferred for css class names versus underscores. Also, .questionNum uses camel case, so this is introducing yet another convention. I recommend deciding on one and aligning all to that convention.